src/App.vue

```diff
@@ -36,14 +59,52 @@ export default {
       beers: null,
       loading: true,
       errored: false,
-      errors: []
+      errors: [],
+      beerSearch: '',
+      beerStrength: 'all'
     };
   },
+  computed: {
+    /**
+     * Filter beers.
+     *
+     * @returns {Array} The filtered beers.
+     */
+    filterBeers() {
+      const filtered = this.beers
+        .filter(beer =>
+          beer.name.toLowerCase().includes(this.beerSearch.toLowerCase())
+        )
+        .filter(beer => {
```

Comment:
This can be improved

1. If else statements are hard to read.
When you use return in the body of your if, the else becomes redundant as it will never be reached. What you wrote is the same as:

```
 if (this.beerStrength == 'weak') {
   return beer.abv <= 4.5;
}
 if (this.beerStrength == 'medium') {
   return beer.abv > 4.5 && beer.abv <= 7.5;
} 
if (this.beerStrength == 'strong') {
   return beer.abv > 7.5;
}
return beer;
```

2. Always use `===` instead of `==`
`===` check for type inequality and will avoid unexpected results. I invite you to read about it as it is not Javascript specific and as a few articles with examples.

3. Using constant for string values
A bit overkill here as you are not reusing the 3 beerStrength value a lot, but if you were, consider using a constant to make it easy to update in case it changes in the future (which happens in real apps). Setting `const LOW_STRENGTH_TYPE = 'weak';` and reusing it makes it more maintainable. 

4. Consider using objects
Sometimes, instead of multiple if, objects can be a good alternative.
For example:
```
const strengths = { weak: beer.adv < 4.5, medium: beer.abv > 4.5 && beer.abv <= 7.5, strong: beer.abv > 7.5, all: beer }

return strengths[this.beerStrength];
```
This is shorter and easier to read, and less code.
It only works if you define the object here as the conditions will be evaluated here though, rather than moved outside of this block. This article explains it well: https://ultimatecourses.com/blog/deprecating-the-switch-statement-for-object-literals

Reply:
So much cleaner, easier to read and much more maintainable 👍